testing/trino-tests/src/test/java/io/trino/tests/TestMinWorkerRequirement.java

```diff
@@ -173,61 +162,4 @@ public void testRequiredWorkerNodesSessionOverride()
             queryRunner.execute(require6Workers, "SELECT COUNT(*) from lineitem");
         }
     }
-
-    @Test
-    public void testMultipleRequiredWorkerNodesSessionOverride()
```

Comment:
I highly suspect that instead of just the test being flaky it's a product issue that the test is uncovering.

Specially because it's not flaky at all on older versions (I checked on 402 only, didn't bisect).

Reply:
for the record, here is the issue https://github.com/trinodb/trino/issues/19446
and here is a potential stop gap solution: https://github.com/trinodb/trino/pull/19653 (admittedly not ideal, but maybe comparable to removing the test)